samples/subsys/usb/cdc_acm/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Source common USB sample options used to initialize new experimental USB
+# device stack. You do not need to source this Kconfig file in your custom
```

Comment:
Here you state that `not need` but in the Kconfig file you state that it `cannot` be used by application. The two comments should be coherent.

Reply:
Changed it, but I am not sure if it is less confusing.

Reply:
Should be fixed now.